tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Test for unsetTtl command.
  */
-public final class UnsetTtlTest extends AbstractAlluxioShellTest {
+public final class UnsetTtlCommandIntegrationTest extends AbstractAlluxioShellTest {
```

Comment:
According to the title of the pull request, the purpose is to change RmrCommandTest to RmrCommandIntegrationTest. This change is unrelated to that, so it should not be included in the pull request.